package.json

```diff
@@ -19,6 +19,7 @@
     "typescript": "^5.3.3"
   },
   "devDependencies": {
-    "@biomejs/biome": "1.6.3"
+    "@biomejs/biome": "1.6.3",
+    "prettier": "^3.2.5"
```

Comment:
I don't have the astro prettier plugin in here. Don't forget to add it.

This may have been the reason why my formatting changed the output of my file unexpectedly...